docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,66 @@
+- Selected features will be labeled “triage/accepted”. Rest will be labelled “triage/ice-boxed” and will be eligible for next planning cycle. Accepted features will be added to the correct milestone and sprint specific project for tracking.
+
+## How to manage Epic/Scenario?
+- Epic/Scenario typically spans 1-3 months or even more in some cases.
+- For new proposals these will be created and triaged/refined just like Story/Feature. However, after triaging these need to be broken down into Features for planning and voting.
+- Github doesn’t allow nested Issues, hence the only good way to track these are by creating Projects and adding the broken down features as issues to this project. An issue can be part of multiple projects and hence we can get a Scenario\Epic view as well as a Sprint view through respective projects.
+
+## Planning criteria:
+- Features are well defined and scoped with clear exit criteria. No feature creeps once release starts
+- Features can be implemented in <2w by one engineer. In case of Epic/Scenario, it could be broken down into <2w features with clear exit criteria.
+- The engineering team should be confident that any engineer can implement the feature based on the content in the feature *(This is similar to refinement in a scrum, but doesn’t have to be as strict as a functional specification)*.
+
+## Organizing with Github Projects 
+- New Project per sprint. Today, the sprint and milestone are both aligned. However in future we may decide to have multiple sprints in a release cycle.
+- New Project per any Story/Scenario
+- Any other projects that help organize and will be created when needed.
+
+## Organizing with Github Milestone 
+- A new milestone is created in Github for each 6 weeks release cycle.
+- All features and bugs that are planned for the release cycle are assigned to the release for tracking the progress of the milestone.
+
+## Pending work and proposed changes:
+- Decide estimation technique (story points on fibonacci, t-shirt size etc)
+- Create features and bugs for 0.6 release.
+- Create appropriate templates and verify if things will work with our current permission model and Prow - I have experimented all this in my private repo, but do not have right permissions in eventing repo. Will need some help here - @Grant?
+- Create following labels:
+    - triage/needstriage
+    - triage/accepted
+    - kind/rejected
+    - triage/ice-boxed
+    - kind/feature
+    - priority/0 - Critical issues that needs to be addressed right away in current release
+    - priority/1 - Highest priority
+    - priority/2
+    - priority/3
+    - priority/4 - lowest priority
+    - severity/1
+    - severity/2
+    - severity/3
+    - severity/4
+- Need to define Priorities and Severities. Is there any norm followed in Knative?
```

Comment:
Do we need both priorities and severities? I've rarely found both to be used effectively, and they usually end up mirror each other.

Reply:
Here are the final list of labels we will need:
    - kind/feature
    - kind/bug
    - priority/0
      - Bugs that need to be fixed in current release, irrespective of when they are created.
    - priority/1
      - Bugs that need to be fixed in the next release.
    - priority/2
     - Bugs that need to be fixed in next 2-3 releases.

- No priority label: Needs triage
- Rejected - Close the GitHub issue
- P2 - Ice-boxed

Reply:
I added `kind/customer-issue`

We also currently have `kind/feature-request`. Do you want me to rename that?

Take a look at the labels in https://github.com/knative/eventing/labels and see if they meet your needs. I think we can probably remove the `/area` ones, too.

Reply:
no need. Lets continue to use kind/feature-request and kind/bug and create the explicitly named ones for priority. I have already updated the document with it and update this PR